internal/cli/cli.go

```
@@ -122,6 +125,10 @@ func runInit(args []string, stdout, stderr io.Writer, dependencies runtimeDepend
 		printInitUsage(stderr)
 		return exitcode.Usage
 	}
+	if values["--evidence-profile"] == "" {
+		fmt.Fprintln(stderr, "AGX-EVIDENCE-PROFILE-REQUIRED")
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)